org.eclipse.buildship.kotlin/build.gradle

```diff
@@ -0,0 +1,21 @@
+buildscript {
+    ext.kotlin_version = '1.0.4'
+    repositories.jcenter()
+    dependencies.classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+}
+
+apply plugin: eclipsebuild.BundlePlugin
+apply plugin: 'kotlin'
+
+dependencies {
+    compile 'org.gradle:gradle-core:3.3'
+    compile 'org.gradle:gradle-process-services:3.3'
```

Comment:
Sorry for the sketchy answer. We need them to compile `KotlinBuildScript.kt` and `GradleKotlinScriptDependenciesResolver.kt`. These classes were copied from `gradle-script-kotlin` which depends on those libraries. Without the dependencies we get the following compile errors:
```
:org.eclipse.buildship.kotlin:compileKotlin
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (11, 23): Unresolved reference: Project
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (12, 23): Unresolved reference: plugins
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (19, 43): Unresolved reference: Project
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (19, 54): Only interfaces can be delegated to
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (19, 54): Unresolved reference: Project
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (28, 49): Unresolved reference: ObjectConfigurationAction
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (29, 9): Unresolved reference: project
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (29, 17): Public-API inline function cannot access non-public-API 'internal open fun <ERROR FUNCTION>(): [ERROR : <ERROR FUNCTION RETURN TYPE>] defined in root package'
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (29, 25): Unresolved reference: it
e: /Development/git/buildship/org.eclipse.buildship.kotlin/src/main/kotlin/org/eclipse/buildship/kotlin/KotlinBuildScript.kt: (29, 28): Public-API inline function cannot access non-public-API 'internal open fun <ERROR FUNCTION>(): [ERROR : <ERROR FUNCTION RETURN TYPE>] defined in root package'
```
I had to copy and modify those classes to make the Kotlin support working under Eclipse. We should discuss with Rodrigo how to share the implementation between Buildship and gradle-script-kotlin.